Frontend/src/data/rental/inputs.js

```diff
@@ -234,6 +257,10 @@ export default {
       component: AutocompleteInput,
       nobind: true,
       props: {
+        localSorting: true,
+        sortByMatchedKeywords: true,
+        itemSortFunction: () => sortItemByIdOrName,
```

Comment:
```suggestion
        itemSortFunction: sortItemByIdOrName,
```
I think it won't work otherwise

Reply:
Mhh strange, for me it was exactly that change that made it work. I'll check again once I'm back home.